### DISCUSSION TITLE
Can't compile for Android

### DISCUSSION THREAD

**Discussion by qacti** [axmolengine/axmol#2197] [Q&A] [answered]
Hello. I'm not able to compile for Android under Arch Linux.

I'm trying to build ```tests/cpp-test``` project with ```axmol -p android -a arm64``` and after some time of building it fails with ```ld: error: unable to find library -lpipewire-0.3```. Same on the newly created project.

I tried to build with SDK/NDK installed on my machine and with one downloaded by axmol but neither of this worked. But if I try to compile some .cpp file with ```-lpipewire-0.3``` I got no errors, so clang can find it.

I found 2 guys with the same problem, one fixed it by removing steam-native from their system, so did I and it doesn't help too (https://github.com/axmolengine/axmol/issues/684). And second guy got ignored (https://github.com/axmolengine/axmol/discussions/1399).

**Comment by halx99** [comment]
does the stream-native installed on your ArchLinux?

**Reply by halx99** [reply, marked as answer]
A fixed PR created: https://github.com/axmolengine/axmol/pull/2200 please check
